handlers/clients.go

```diff
@@ -0,0 +1,25 @@
+package handlers
+
+import (
+	"context"
+	"net/url"
+
+	searchC "github.com/ONSdigital/dp-api-clients-go/site-search"
+)
+
+//go:generate moq -out clients_mock.go -pkg handlers . RenderClient SearchClient
+
+// ClientError is an interface that can be used to retrieve the status code if a client has errored
+type ClientError interface {
+	Code() int
```

Comment:
I had a look at the handlers_test and it mocks this interface but includes the Error() method, which seems to be missing from here? 
```suggestion
	Error() string
	Code() int
```

Reply:
We are only using the `ClientError` (without mocking) in the function `setStatusCode` (https://github.com/ONSdigital/dp-frontend-search-controller/blob/develop/handlers/handlers.go#L165). In that function, we are using `ClientError` to map the error to get the `Code` of the error. We don't care about the Error() when it comes to the interface itself.

Reply:
This was discussed in the PR (https://github.com/ONSdigital/dp-frontend-search-controller/pull/23). I have unresolved the particular discussion so you can see it straight away